messages/create_scratch.md

```diff
@@ -70,6 +74,10 @@ Consumer key of the Dev Hub connected app.
 
 Number of minutes to wait for the scratch org to be ready.
 
+# flags.wait.description
+
+If the command continues to run after the wait period, the CLI returns control of the terminal to you and displays the job ID. To resume the scratch org creation, run "sf env resume scratch".
```

Comment:
Since it requires either the `--job-id` or the `--use-most-recent`, would it make more sense to say, "run the env resume scratch command" rather than giving them something they'll try to copy/paste unsuccessfully?

Reply:
Done.